src/gamestates/TestState.cpp

```diff
@@ -2,12 +2,12 @@
 
 #include "TestState.h"
 
-#include "GameObjectMediator.h"
-#include "Universe.h"
-#include "CollisionDetector.h"
-#include "BasicEnemy.h"
-#include "GraphicsHandler.h"
-#include "MainCharacter.h"
+#include "src/GameObjectMediator.h"
```

Comment:
Not sure :/, tried to look into it b4 but things I tried didn't work. Tried a big combination of things with the cmake and the file includes. Il give it one more shot though